### PR TITLE
[DC-2457] Replace all bq.get_client calls in data_steward/analytics

### PR DESCRIPTION
**Pull request by lledezma**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/ad_hoc_analyses/ac120_cpt_modifiers.py [hunk 1]
```diff
@@ -74,7 +55,8 @@ def cstr(s, color='black'):
 
 
 '''.format(DATASET_ID=dataset_id)
-number_of_rows_affected = pd.read_gbq(number_of_rows_affected,dialect="standard")
+number_of_rows_affected = pd.read_gbq(number_of_rows_affected,
+                                      dialect="standard")
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/ad_hoc_analyses/ac120_cpt_modifiers.py:59]: Are these formatting changes due to yapf?

**Review by nishanthpp93 — COMMENTED** [review]
Overall looks good. Added some comments to clarify.

File: data_steward/analytics/cdr_ops/ad_hoc_analyses/ac120_cpt_modifiers.py [hunk 2]
```diff
@@ -23,38 +23,19 @@
...
-import matplotlib
 import matplotlib.pyplot as plt
-import matplotlib.lines as mlines
-from matplotlib.lines import Line2D
-
-import matplotlib.ticker as ticker
-import matplotlib.cm as cm
-import matplotlib as mpl
 
 import matplotlib.pyplot as plt
```
[the remark this answers is not in the document]
Comment (lledezma) [reply data_steward/analytics/cdr_ops/ad_hoc_analyses/ac120_cpt_modifiers.py:28]: Yes. I found many unused imports.

File: data_steward/analytics/cdr_ops/ad_hoc_analyses/ac120_cpt_modifiers.py (hunk 1 above)
Comment (lledezma) [reply data_steward/analytics/cdr_ops/ad_hoc_analyses/ac120_cpt_modifiers.py:59]: Yes. Most of the changes were coming yapf.

**Pull request merged**